runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -630,76 +630,84 @@ function! s:OpenHoverPreview(lines, filetype) abort
 
     let use_float_win = s:ShouldUseFloatWindow()
     if use_float_win
-        let bufname = nvim_create_buf(v:false, v:true)
-        call nvim_buf_set_lines(buf, 0, -1, v:true, lines)
-        let pos = getpos('.')
-
-        " Calculate width and height and give margin to lines
-        let width = 0
-        for index in range(len(lines))
-            let line = lines[index]
-            if line !=# ''
-                " Give a left margin
-                let line = ' ' . line
-            endif
-            let lw = strdisplaywidth(line)
-            if lw > width
-                let width = lw
-            endif
-            let lines[index] = line
-        endfor
-
-        " Give margin
-        let width += 1
-        let lines = [''] + lines + ['']
-        let height = len(lines)
-
-        " Calculate anchor
-        " Prefer North, but if there is no space, fallback into South
-        let bottom_line = line('w0') + winheight(0) - 1
-        if pos[1] + height <= bottom_line
-            let vert = 'N'
-            let row = 1
-        else
-            let vert = 'S'
-            let row = 0
+      let bufname = nvim_create_buf(v:false, v:true)
```

Comment:
This is a number, not name, right?  Maybe `bufhandle` is a better name.

Reply:
or just `buf`.